content/en/design/core/_index.md

```diff
@@ -73,152 +55,64 @@ These are the status indication colors of the system. When necessary, use white
 
 For more information on how these colors are applied in the app, see our [color variables file](https://github.com/medic/medic/blob/master/webapp/src/css/variables.less). 
 
----
 
 ## Typography
 
-The default app font is [Noto Sans](https://www.google.com/get/noto/). It is free, open source, supports 800 languages and was specifically designed for good web legibility. It is bundled with the app so that all users see the same font regardless of their particular device, language, browser, etc. This ensures a consistent experience for all users.
-
-**Type size:** A standard minimum size for web content is 14-16 points/pixels.
-
-**Type case:** Lowercase letters are easier to read than text set in all-caps.
-
-**Line spacing:** A general rule is that the line spacing value should be anything between 1.25 and 1.5 times greater than the font size.
+The default app font is <a href="https://www.google.com/get/noto/">Noto Sans</a>. It is free, open source, supports 800 languages and was specifically designed for good web legibility. It is bundled with the app so that all users see the same font regardless of their particular device, language, browser, etc. This ensures a consistent experience for all users.
 
-**Line length:** The optimal line length for body text is considered to be 50-60 characters per line on desktop and 30-40 on mobile, including spaces.
-
-**Alignment:** Text in the app should be left-aligned by default unless otherwise specified. 
-
-**Color:** Most text in the app should be the ```@text-normal-color: @gray-ultra-dark color```.
+Most text in the app should be the ```@text-normal-color: @gray-ultra-dark color```.
 The lighter text color ```@text-secondary-color: @gray-dark)``` is used for labels and condition card filters.
 Hyperlinked text color is ```@text-hyperlink-color: @blue-dark)```. 
```

Comment:
Are these meant to be inline code, surrounded by a single backtick?

Reply:
Yep..